modules/govuk_rabbitmq/manifests/consumer.pp

```diff
@@ -48,9 +61,9 @@
       ensure => $ensure,
       type   => $exchange_type,
     }
-    $write_permission = "^(amq\\.gen.*|${amqp_queue}|${amqp_exchange})$"
+    $write_permission = "^(amq\\.gen.*|${amqp_queue}|${amqp_exchange})\$"
```

Comment:
https://docs.puppet.com/puppet/3.8/reference/future_lang_data_string.html#escape-sequences

It's to explicitly include the $ at the end in the permissions regex. It worked before, but $ should be escaped if not being used for interpolation AIUI. My IDE inspections agree...